Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -269,7 +269,7 @@ extension LoginListViewController: LoginDataSourceObserver {
         loadingStateView.isHidden = true
         tableView.reloadData()
         activeLoginQuery = nil
-        navigationItem.rightBarButtonItem?.isEnabled = loginDataSource.count > 0
+        navigationItem.rightBarButtonItem?.isEnabled = loginDataSource.count > 0 // swiftlint:disable:this empty_count
```

Comment:
Two concerns about this rule in general
1. This basically prevents us from declaring local `count` variables and custom `count` methods
2. I feel like `!logins.isEmpty` is less readable than `logins.count > 0`

Thoughts on both points?
cc @kylehickinson @Brandon-T @soner-yuksel

Reply:
The project had the rule configured to throw an error but the rule was disable. This was causing SwiftLint to throw a warning stating the rule was configured but disabled.

It enabling the rule isn't desired, I could update this PR reverting these changes and removing the configuration.

Reply:
Thanks for your PR regardless @otaviocc it looks nice overall :)

But since we are at swiftlint update task we can always rethink if this rule is worth it,
for context, we originally took the whole file from Firefox, then modified it slightly
https://github.com/mozilla-mobile/firefox-ios/blob/main/.swiftlint.yml

Reply:
The only case where `isEmpty` has an advantage over `count` is Swift Strings. Otherwise there is no benefit of using `isEmpty` under the hood. I believe we do not need to have a lint rule for this. 
I particularly do not like to have rules If they are necessary like disabling force-unwrapping.

Reply:
I personally prefer to use `isEmpty` when I can, but I don't have any strong opinions on a linting rule to enforce it. Fine with disabling it and removing the configuration of it to silence the warnings about it but also fine with keeping a majority of the `count` → `isEmpty` changes @otaviocc already made

Reply:
I feel like we should not enforce the `count` lint. Remove it completely in my opinion. Any developer that knows Swift should automatically attempt `.isEmpty` first OR `.count > 0` or just know that `.isEmpty` is for Strings and Arrays by default. Forcing it and then having to disable it everywhere there is a custom property or method doesn't make much sense to me. I don't mind the comment one. That should be forced for consistency.